freqtrade/freqtradebot.py

```diff
@@ -957,11 +997,17 @@ def handle_timedout_limit_sell(self, trade: Trade, order: Dict) -> str:
             trade.close_date = None
             trade.is_open = True
             trade.open_order_id = None
+        else:
+            # TODO: figure out how to handle partially complete sell orders
+            reason = CancelReason.PARTIALLY_FILLED.value
```

Comment:
Note to self: we need to handle this differently and/or prevent notification spamming (probably as a first step)

If an order get's stuck here it'll send "not cancelled" over and over again (every iteration ...).

Leave as is for now - it's not making it worse - and this is complex enough :)